envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/resource/listeners/filters/RBACFilterFactory.kt

```diff
@@ -291,8 +298,21 @@ class RBACFilterFactory(
         val clients = incomingEndpoint.clients.flatMap { clientOrRole ->
             roles.find { it.name == clientOrRole.name }?.clients ?: setOf(clientOrRole)
         }
+        val clientsWithDefaults = mutableListOf<ClientWithSelector>()
```

Comment:
I would name that variable `clientsWithPredefinedList`, because it could be merged with custom list (not default one).

Reply:
changed